applications/salk-portal/src/components/home/DeleteExperimentDialog.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from "react";
+import {
+  makeStyles,
+} from "@material-ui/core";
+import { Button, Dialog, DialogActions, DialogContent,  DialogTitle, IconButton, Box, Typography } from "@material-ui/core";
+import {orange} from  "../../theme"
+import CLOSE from "../../assets/images/icons/close.svg";
+
+const useStyles = makeStyles(() => ({
+    info: {
+        '& .MuiTypography-root': {
+            fontSize: '0.75rem',
+            lineHeight: '1rem',
+            letterSpacing: '0.001em',
+          },
+    }, 
+
+}));
+
+
+
+interface IDeleteDialog {
+    open:boolean;
+    handleClose: () => void
+}
+
+export const DeleteExperimentDialog: React.FC<IDeleteDialog> = ({open, handleClose}) => {
+  const classes = useStyles()
+
+  return (
+    <Dialog
+      fullWidth={true}
+      maxWidth='xs'
+      open={open}
+      onClose={handleClose}
+    >
+      <DialogTitle>
```

Comment:
@paularah when you inspect you would notice MuiDialogTitle-root - font-weight, line-height, color is not correct
MuiDialogContent-root - Letter spacing issue, color not added

Reply:
@paularah you could even message Paul Faivret and ask him if we can keep similar styles for both the modals